### ISSUE TITLE
8.0.1 package version showing as latest in npm, rather than 9.0.0

### ISSUE THREAD

**Issue by greg-murray-volusion** [mikaelbr/node-notifier#355]
I see you published the package yesterday, but for some reason [npm is not showing 9.0.0 as the latest](https://www.npmjs.com/package/node-notifier)

```bash
npm view node-notifier

node-notifier@8.0.1 | MIT | deps: 6 | versions: 67
A Node.js module for sending notifications on native Mac, Windows (post and pre 8) and Linux (or Growl as fallback)
https://github.com/mikaelbr/node-notifier#readme

keywords: notification center, mac os x 10.8, notify, terminal-notifier, notify-send, growl, windows 8 notification, toaster, notification

dist
.tarball: https://volusion.jfrog.io/artifactory/api/npm/npm/node-notifier/-/node-notifier-8.0.1.tgz
.shasum: f86e89bbc925f2b068784b31f382afdc6ca56be1

dependencies:
growly: ^1.3.0     semver: ^7.3.2     uuid: ^8.3.0       
is-wsl: ^2.2.0     shellwords: ^0.1.1 which: ^2.0.2      

maintainers:
- mikaelb <mikaelbre@gmail.com>

dist-tags:
latest: 8.0.1  

published yesterday by mikaelb <mikaelbre@gmail.com>
```

Trying to address npm audit failure for this [advisory](https://snyk.io/vuln/SNYK-JS-NODENOTIFIER-1035794)

**Comment by mikaelbr** [comment]
Oh! Sorry about that. I updated the dist-tag now, so 9.0.0 should again be latest. I published a 8.0.1 patch a couple of days ago with only the security patch without breaking changes so a bump in lock files would fix the security issue. But by doing so  npm changed the latest tag.

**Comment by greg-murray-volusion** [comment]
Great, thanks so much!
